### ISSUE TITLE
ec2_group should allow for security group revocations

### ISSUE THREAD

**Issue by Sodki** [ansible/ansible-modules-core#4251]
##### ISSUE TYPE
- Feature Idea
##### COMPONENT NAME

ec2_group
##### ANSIBLE VERSION

ansible 2.1.0.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = Default w/o overrides
##### OS / ENVIRONMENT

N/A
##### SUMMARY

ec2_group allows us to create security groups, to remove security groups and to add new rules to security groups. What it doesn't allow us to do is to remove rules, so we can't easily revoke rules that have been set. This feature is already available in AWS and boto3 ([relevant documentation](https://boto3.readthedocs.io/en/latest/reference/services/ec2.html#EC2.SecurityGroup.revoke_ingress).
##### STEPS TO REPRODUCE

Let's say you have a playbook that launches a new EC2 instance. That role also adds a new rule to an existing ELB, so that the new EC2 instance can access it. If you now have a `cleanup` playbook that destroys that EC2 instance, you ideally want to remove the rule you just added to the ELB. A simple revoke would be ideal, but that feature is not available.

We could have two new options to the module, `revoke_rules` and `revoke_rules_egress`, to take care of this.

**Comment by ansibot** [comment]
@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/jctanner/ansibullbot/blob/ISSUE_TRIAGE_CLASS/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by ansibot** [comment]
This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by Sodki** [comment]
Superseded by ansible/ansible#20456.
